lib/spack/docs/features.rst

```diff
@@ -1,4 +1,4 @@
-================
+ ================
```

Comment:
I don't think this space should be here.

Reply:
This space is causing the documentation tests to fail because Sphinx doesn't like it.